tests/utils/coordinates.test.js

```diff
@@ -24,7 +24,7 @@ suite('utils.coordinates', function () {
     });
 
     test('parses null', function () {
-      assert.equal(coordinates.parse(null), null);
+      assert.equal(coordinates.parse(null), undefined);
```

Comment:
I believe that returning `null` on `null` input is sensible. `parse` behaves as a pass through if the value is not parsable.